modules/gitops-accessing-gitops-monitoring-dashboards.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/monitoring-with-gitops-dashboards.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-accessing-gitops-monitoring-dashboards_{context}"]
+= Accessing {gitops-shortname} monitoring dashboards
+
+The monitoring dashboards are deployed automatically by the Operator. You can access {gitops-shortname} monitoring dashboards from the *Administrator* perspective of the {OCP} web console. 
+
+[NOTE]
+====
+Disabling or changing the content of the dashboards is not supported.
+====
+
+.Prerequisites
+
+* You have access to the {OCP} web console.
+* The {gitops-title} Operator is installed in the default namespace, `openshift-gitops-operator`.
+* The cluster monitoring is enabled on the `openshift-gitops-operator` namespace.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, go to *Observe* → *Dashboards*.
+
+. From the *Dashboard* drop-down list, select the desired {gitops-shortname} dashboard: *{gitops-shortname} (Overview)*, *{gitops-shortname} / Components*, or *{gitops-shortname} /  gRPC Services*.
+
+. Optional: Choose specific namespace, cluster, and interval from the *Namespace*, *Cluster*, and *Interval* drop-down lists.
```

Comment:
```suggestion
. Optional: Choose specific a namespace, cluster, and interval from the *Namespace*, *Cluster*, and *Interval* drop-down lists.
```

Reply:
Ha, I just realized how silly this recommendation-typo was. 😬 Hopefully the intent of putting an "a" in the correct spot came through either way. 😅